benches/bench_eq.rs

```diff
@@ -27,5 +44,39 @@ pub fn eq_eval(c: &mut Criterion) {
     });
 }
 
-criterion_group!(bench_eq, eq_keygen, eq_eval);
+pub fn eq_batch_eval(c: &mut Criterion) {
+    let mut keys_a: Vec<u8> = Vec::with_capacity(3105);
+    let mut keys_b: Vec<u8> = Vec::with_capacity(3105);
+    let keys_a_pointer: *mut u8 = keys_a.as_mut_ptr();
+    let keys_b_pointer: *mut u8 = keys_b.as_mut_ptr();
+    let n_values: usize = 5;
+    let n_threads: usize = 6;
```

Comment:
I don't know how many threads are going to be used by the Github Actions worker. Anyway, it is probably worth launching multiple benches with n_threads = 1, 4, 6 for instance

Reply:
[Here](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) is some info on the specifications of the GitHub Actions runners. I don't know how many threads per core they have but we could certainly add the benchmarks.